PraceDomowe/PracaDomowa5/Nahorny_Jakub/Praca domowa 5.ipynb

```diff
@@ -0,0 +1,1763 @@
+{
```

Comment:
<p>To nie jest do końca dobrze, zliczasz posty które posiadają AcceptedAnswer, a powinieneś zliczać posty które są oznaczone jako AcceptedAnswer dla innego postu.</p><p> </p><p><code>0.5 punktu</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-ExploratoryDataAnalysis/pull/159/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>